.devcontainer/Dockerfile

```diff
@@ -1,10 +1,18 @@
-FROM ghcr.io/nunit/docfx-action:v2.9.0
+FROM ghcr.io/nunit/docfx-action:v2.10.0
 EXPOSE 8080
+ARG NODE_MAJOR=20
 
 ### Installing Node LTS into the container
-RUN curl -fsSL https://deb.nodesource.com/setup_lts.x | bash -
-RUN apt-get install nodejs
+RUN apt-get update
```

Comment:
ℹ️ Added this based on what I saw in the build output. I used it during my PR review so I know it works 👍